src/constants/SharedPropTypes.js

```diff
@@ -4,3 +4,12 @@ export const OverridePropTypes = PropTypes.shape({
   props: PropTypes.object,
   component: PropTypes.elementType,
 });
+
+export const ListTagsPropTypes = PropTypes.arrayOf(
+  PropTypes.shape({
+    /** The content of the tag */
+    content: PropTypes.string,
+    /** The type (normally color, see Tag component) */
+    type: PropTypes.string,
```

Comment:
I would be okay with this being `PropTypes.arrayOf(PropTypes.node)` instead of an object structure so we won't have to track and update the structure to match `Tag` prop names when/if they change or are added to. This would also allow folks to use the Tag skeleton if the tag content is async/loading, as well as the additional `disabled` and `title` props available on Tag.

The example in storybook and the documentation of the `tags` prop specify that the intention is for an array of `Tag`s to be passed in.

Reply:
Yes, good idea.